### PR TITLE
bump cri-api to k8s 1.25 rc0

### PR DESCRIPTION
**Pull request by harche**
Signed-off-by: Harshal Patil <harpatil@redhat.com>

#### What type of PR is this?

/kind api-change

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by harche** [comment]
Required by - https://github.com/cri-o/cri-o/pull/6094/

File: server/cri/v1/rpc_container_events.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package v1
+
+import (
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func (s *service) GetContainerEvents(req *pb.GetEventsRequest, ces pb.RuntimeService_GetContainerEventsServer) error {
```
Comment (haircommander) [inline server/cri/v1/rpc_container_events.go:7]: nit: I find it odd we don't have a corresponding comment
`	// Spoofing the GetContainerEvents method to return nil response until actual support code is added.`

Comment (harche) [reply server/cri/v1/rpc_container_events.go:7]: Done.

**Comment by haircommander** [comment]
/lgtm
/approve

thanks!

**Pull request merged**